Editor/Editors/CinemachineVirtualCameraEditor.cs

```diff
@@ -120,6 +126,95 @@ private void OnSceneGUI()
                 InspectorUtility.RepaintGameView();
                 Target.m_UserIsDragging = false;
             }
+
+            base.OnSceneGUI();
+            m_PipelineSet.OnSceneGUI(); // call hidden editors
+        }
+        
+        protected override void DrawSceneTools()
+        {
+            var vcam = Target;
+            if (!vcam.IsValid || vcam.m_ExcludedPropertiesInInspector.Contains("m_Lens"))
```

Comment:
I used `vcam.m_ExcludedPropertiesInInspector.Contains("m_Lens")`, because this way there is no need to introduce a variable. What do you think?